index.html

```diff
@@ -69,6 +78,242 @@ <h1 class="primary-txt">
                 </ul>
             </div>
         </section>
+        <section class="works">
+            <div class="card-works">
+                <div class="snapshoot-portfolio bg-1">
+                </div>
+                <div class="left-block">
+                    <div>
+                        <h1 class="primary-txt">
+                            Tonic
+                        </h1>
+                        <div class="frame">
+                            <div class="client">
+                                <h4 class="frame-client-text">CANOPY</h4>
+                            </div>
+                            <ul class="frame-client-detail">
+                                <li>
+                                    <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                        <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
+                                    </svg>
+                                    Back End Dev
+                                </li>
+                                <li>
+                                    <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                        <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
+                                    </svg>
+                                    2015
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+                    <p class="detail-txt">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <div class="tags">
+                        <div>
+                            <p><span class="tag">html</span>
+
+                            </p>
+                        </div>
+                        <div>
+                            <p>
+                                <span class="tag">css</span>
+                            </p>
+                        </div>
+                        <div>
+                            <p>
+                                <span class="tag">javascript</span>
+                            </p>
+                        </div>
+                    </div>
```

Comment:
- [ ] In order to improve sematics, kindly make use of the `ul` and `li` tags in this section.